src/v/raft/consensus.cc

```diff
@@ -2114,7 +2142,7 @@ ss::future<> consensus::write_snapshot(write_snapshot_cfg cfg) {
 ss::future<>
 consensus::do_write_snapshot(model::offset last_included_index, iobuf&& data) {
     vlog(
-      _ctxlog.trace,
+      _ctxlog.info,
```

Comment:
is that change inteded ?

Reply:
kind of... we are already logging a bunch of stuff on log prefix truncation, so I thought additional logging won't hurt